cli/lib/config.js

```diff
@@ -2,35 +2,37 @@
 
 module.exports = { load, save };
 
-const { promises: fs } = require('fs');
+const fs = require('fs');
 const toml = require('@iarna/toml');
 const home = require('user-home');
 const path = require('path');
 
 const errors = require('./errors');
 
 async function load(filename = path.join(home, '.entropicrc')) {
-  var content = null;
-  try {
-    content = await fs.readFile(filename, 'utf8');
-  } catch (e) {
-    if (e.code === 'ENOENT') {
-      return {};
-    }
-
-    throw new errors.CouldNotReadConfigFile(filename, e);
-  }
-
-  var parsed = null;
-  try {
-    parsed = toml.parse(content);
-  } catch (e) {
-    throw new errors.CouldNotParseConfigToml(filename, e);
-  }
+  var content = '';
+  var parsed = '';
+  await fs.readFile(filename, 'utf8', (e, data) => {    
+    if (e.code === 'ENOENT') {      
+      if (!fs.existsSync(filename)){
+        fs.mkdirSync(filename);
```

Comment:
Why do you need to make directory?

Reply:
Suppose to be the dir for .toml files 🤔

Reply:
I see. Good intention, but as of now I see only one file in the home directory to be created + one in the root of the project.